src/schema/v2/sorts/collection_sorts.ts

```diff
@@ -9,6 +9,12 @@ const CollectionSorts = new GraphQLEnumType({
     POSITION_DESC: {
       value: "-position",
     },
+    SAVED_AT_ASC: {
+      value: "created_at",
+    },
+    SAVED_AT_DESC: {
+      value: "-created_at",
+    },
```

Comment:
These are named `created_at` in Gravity, but per our ticket I've renamed these to `SAVED_AT` to better reflect their meaning. 

(An alternative would have been to place a `created_at` timestamp on each `ArtworkEdge`, which would reflect when the Gravity `CollectedArtwork` instance was created, i.e. when the artwork was saved — but we don't have any immediate need for that.)